### ISSUE TITLE
Cannot publish plugin

### ISSUE THREAD

**Issue by fredericlb** [c9/core#187]
When trying to (first) publish my plugin using the c9 utility, I have the following output : 

```sh
fredericlb@c9-extended-navigation:~/.c9/plugins/extended-navigation (master) $ c9 publish --verbose --force 0.0.1
wrapping module plugins/extended-navigation/ext/gonzales.scss.js
wrapping module plugins/extended-navigation/ext/gonzales.sass.js
wrapping module plugins/extended-navigation/ext/gonzales.less.js
wrapping module plugins/extended-navigation/ext/gonzales.css.js
wrapping module plugins/extended-navigation/ext/gonzales.core.js
wrapping module plugins/extended-navigation/ext/acorn_loose.js
wrapping module plugins/extended-navigation/ext/acorn.js
wrapping module plugins/extended-navigation/ext/acorn
Processing 16 files.
Compressing:  true
Adding '/home/ubuntu/.c9/plugins/extended-navigation/ext/xhtml.js'.
Adding '/home/ubuntu/.c9/plugins/extended-navigation/ext/acorn-jsx.js'.
Adding '/home/ubuntu/.c9/plugins/extended-navigation/ext/acorn.js'.
Adding '/home/ubuntu/.c9/plugins/extended-navigation/ext/acorn.js'.
Adding '/home/ubuntu/.c9/plugins/extended-navigation/ext/acorn_loose.js'.
Adding '/home/ubuntu/.c9/plugins/extended-navigation/src/expand.walk.js'.
Adding '/home/ubuntu/.c9/plugins/extended-navigation/src/expand.js.js'.
Adding '/home/ubuntu/.c9/plugins/extended-navigation/ext/gonzales.core.js'.
fredericlb@c9-extended-navigation:~/.c9/plugins/extended-navigation (master) $ c9 publish 0.0.1
wrapping module plugins/extended-navigation/ext/gonzales.scss.js
wrapping module plugins/extended-navigation/ext/gonzales.sass.js
wrapping module plugins/extended-navigation/ext/gonzales.less.js
wrapping module plugins/extended-navigation/ext/gonzales.css.js
wrapping module plugins/extended-navigation/ext/gonzales.core.js
wrapping module plugins/extended-navigation/ext/acorn_loose.js
wrapping module plugins/extended-navigation/ext/acorn.js
wrapping module plugins/extended-navigation/ext/acorn
Processing 16 files.
Compressing:  true
Adding '/home/ubuntu/.c9/plugins/extended-navigation/ext/xhtml.js'.
Adding '/home/ubuntu/.c9/plugins/extended-navigation/ext/acorn-jsx.js'.
Adding '/home/ubuntu/.c9/plugins/extended-navigation/ext/acorn.js'.
Adding '/home/ubuntu/.c9/plugins/extended-navigation/ext/acorn.js'.
Adding '/home/ubuntu/.c9/plugins/extended-navigation/ext/acorn_loose.js'.
Adding '/home/ubuntu/.c9/plugins/extended-navigation/src/expand.walk.js'.
Adding '/home/ubuntu/.c9/plugins/extended-navigation/src/expand.js.js'.
Adding '/home/ubuntu/.c9/plugins/extended-navigation/ext/gonzales.core.js'.
```

No warnings or errors, the process just stops there and I can't find any useful debug output. 

Code can be found in the following repo : https://github.com/fredericlb/c9-extended-navigation

**Comment by nightwing** [comment]
this should be fixed with https://github.com/c9/core/commit/68046d4445529db1e19c5c46a18e408c04bdbd4d 
try updating c9 with `npm install c9 -g;`

**Comment by fredericlb** [comment]
Hi,

nothing changed :( but might be an issue with the c9 package in npm as c9 exec version is the same as before (3.0.2112)

**Comment by nightwing** [comment]
was there an error when running `npm install c9 -g` ? the fixed version is `3.1.126` https://www.npmjs.com/package/c9

**Comment by fredericlb** [comment]
It was an issue with my path to c9 binary ! Thanks it works great now
